lib/LaTeXImage.pm

```diff
@@ -130,22 +145,27 @@ sub header {
 	my @xcolorOpts = grep { ref $_ eq "ARRAY" && $_->[0] eq "xcolor" && defined $_->[1] } @{$self->texPackages};
 	my $xcolorOpts = @xcolorOpts ? $xcolorOpts[0][1] : 'svgnames';
 	push(@output, "\\usepackage[$xcolorOpts]{xcolor}\n");
-	push(@output, "\\usepackage{tikz}\n");
+	# Load tikz if environment is tikzpicture, but not if texPackages contains tikz already
+	my %istikzused = map {ref $_ eq "ARRAY" ? ($_->[0] => ($_->[0] eq 'tikz')) : ($_ => ($_ eq 'tikz'))} @{$self->texPackages};
+	push(@output, "\\usepackage{tikz}\n") if ($self->environment->[0] eq 'tikzpicture' && !$istikzused{'tikz'});
```

Comment:
Here is what I have for this.  This is a somewhat common way of checking a perl array for a specific value.  Your approach is okay though.
```suggestion
	push(@output, "\\usepackage{tikz}\n")
	if ($self->environment->[0] eq 'tikzpicture' &&
		!grep { (ref $_ eq "ARRAY" && $_->[0] eq 'tikz') || $_ eq 'tikz' } @{$self->texPackages});
```